lib/features/notes/presentation/widgets/daily_reminders.dart

```diff
@@ -0,0 +1,154 @@
+import 'package:dairy_app/app/themes/theme_extensions/note_create_page_theme_extensions.dart';
+import 'package:dairy_app/app/themes/theme_extensions/settings_page_theme_extensions.dart';
+import 'package:dairy_app/core/dependency_injection/injection_container.dart';
+import 'package:dairy_app/core/utils/utils.dart';
+import 'package:dairy_app/features/auth/core/constants.dart';
+import 'package:dairy_app/features/auth/data/models/user_config_model.dart';
+import 'package:dairy_app/features/auth/presentation/bloc/user_config/user_config_cubit.dart';
+import 'package:dairy_app/features/notes/domain/repositories/notifications_repository.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:simple_accordion/widgets/AccordionHeaderItem.dart';
+import 'package:simple_accordion/widgets/AccordionItem.dart';
+import 'package:simple_accordion/widgets/AccordionWidget.dart';
+
+// ignore: must_be_immutable
+class DailyReminders extends StatelessWidget {
+  late INotificationsRepository notificationsRepository;
+
+  DailyReminders({Key? key}) : super(key: key) {
+    notificationsRepository = sl<INotificationsRepository>();
+  }
+
+  String getSubtitle(bool? isDailyReminderEnabled, TimeOfDay? reminderTime) {
+    if (isDailyReminderEnabled == null || isDailyReminderEnabled == false) {
+      return "Notifications are not enabled";
+    }
+
+    if (reminderTime == null) {
+      return "You haven't selected a notification time";
+    }
+
+    return "You will be notified at ${UserConfigModel.getTimeOfDayToString(reminderTime)}";
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final userConfigCubit = BlocProvider.of<UserConfigCubit>(context);
+
```

Comment:
Please add this here
```suggestion

final notificationsRepository = sl<INotificationsRepository>();
```